tutorials/documentation_materials/classiq_101/classiq_concepts/execute/advanced_execution/advanced_execution_user_guide.ipynb

```diff
@@ -0,0 +1,834 @@
+{
```

Comment:
<p>The expectation value is of a specific operator <strong>for</strong> the quantum program.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>I didn't understand the note, so I tried to clarify things.</p>